core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -53,6 +54,17 @@ class LocalEntityProvider(workspace: Workspace, implicit protected val dataSourc
             }
             //Else return the full query results
             else {
+              val missReason = if (!cacheEnabled)
+                "cache disabled at system level"
+              else if (!useCache)
+                "user request specified cache bypass"
+              else if (!isEntityCacheCurrent)
+                "cache is out of date"
+              else
+                "unknown reason - this should be unreachable"
+
+              logger.info(s"entity statistics cache: miss ($missReason) [${workspaceContext.workspaceIdAsUUID}]")
+
```

Comment:
the extra logging in this file isn't necessary for this ticket, but I think it will help us analyze performance going forward.